utils/pkg/pkg_run

```diff
@@ -63,12 +72,43 @@ docker_run () {
       fi
     elif [[ $platform_family =~ centos$ ]]; then
       pkgtype="rpm"
+      pkgarch="x86_64"
+      export package_cloud_os="el"
       if [[ $platform_release == "7" ]]; then
         build_docker_image="solarce/centos7-ruby"
         test_docker_image="2k0ri/centos7-systemd"
       fi
     fi
 
+    if [[ $VERSION =~ dev ]]; then
+      export PC_REPO="worker-testing"
+    else
+      export PC_REPO="worker"
+    fi
+
+    if [[ $pkgtype == "rpm" ]]; then
+      PC_VERSION=$(echo $VERSION|sed 's/\-/\_/')
+    else
+      PC_VERSION=$VERSION
+    fi
+
+    export LATEST_PUBLISHED_FILENAME=$(curl -s https://$(echo $PACKAGECLOUD_TOKEN):@packagecloud.io/api/v1/repos/$PC_USER/$PC_REPO/package/$pkgtype/$package_cloud_os/$platform_release/travis-worker/$pkgarch/$PC_VERSION/1.json|jq .'filename'|tr -d '"')
```

Comment:
I'm curious: why the `$(echo $PACKAGECLOUD_TOKEN)` as opposed to using `$PACKAGECLOUD_TOKEN` directly?